### PR TITLE
Duckling extra volume mounts

### PR DESCRIPTION
**Pull request by erohmensing**

### CODE REVIEW

File: charts/rasa-x/templates/duckling-deployment.yaml [hunk 1]
```diff
@@ -59,4 +59,12 @@ spec:
         resources:
           {{- toYaml . | nindent 10 }}
         {{- end }}
+        {{ if .extraVolumeMounts }}
+        volumeMounts:
+{{ toYaml .extraVolumeMounts | indent 8 }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/duckling-deployment.yaml:64]:
```suggestion
{{ toYaml .Values.duckling.extraVolumeMounts | indent 8 }}
```

File: charts/rasa-x/templates/duckling-deployment.yaml [hunk 2]
```diff
@@ -59,4 +59,12 @@ spec:
...
+        {{- end }}
+      {{ if .extraVolumes }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/duckling-deployment.yaml:66]:
```suggestion
      {{ if .Values.duckling.extraVolumes }}
```

File: charts/rasa-x/templates/duckling-deployment.yaml [hunk 3]
```diff
@@ -59,4 +59,12 @@ spec:
...
+      volumes:
+{{ toYaml .extraVolumes | indent 6 }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/duckling-deployment.yaml:68]:
```suggestion
{{ toYaml .Values.duckling.extraVolumes | indent 6 }}
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]
I've left some comments 😄

**Comment by erohmensing** [comment]
Woof 🤦 this is what i get for pushing on a late friday afternoon. Thanks for the thorough review 😄

File: charts/rasa-x/templates/duckling-deployment.yaml [hunk 4]
```diff
@@ -59,4 +59,12 @@ spec:
         resources:
           {{- toYaml . | nindent 10 }}
         {{- end }}
+        {{ if .extraVolumeMounts }}
```
Comment (erohmensing) [inline charts/rasa-x/templates/duckling-deployment.yaml:62]:
```suggestion
        {{ if .Values.duckling.extraVolumeMounts }}
```

**Comment by erohmensing** [comment]
@tczekajlo I've been having to restart the CI a lot to get the logging in to work, is that just something we have to expect at the moment?

**Review by tczekajlo — APPROVED** [review]
LGTM 🚀

**Comment by tczekajlo** [comment]
> @tczekajlo I've been having to restart the CI a lot to get the logging in to work, is that just something we have to expect at the moment?

the CI looks good to me and green 🤔

**Comment by erohmensing** [comment]
Yeah, i re-ran it 3 times though 😅

**Pull request merged**